pkg/controller/schedule_controller.go

```diff
@@ -365,14 +365,18 @@ func getNextRunTime(schedule *api.Schedule, cronSchedule cron.Schedule, asOf tim
 }
 
 func getBackup(item *api.Schedule, timestamp time.Time) *api.Backup {
+	labels := item.Labels
```

Comment:
Please add a comment indicating that this is copying labels from the schedule to the backup + adding the ark-schedule label